src/Tizen.NUI/src/public/ImfManager.cs

```diff
@@ -49,8 +49,53 @@ public static ImfManager Instance
         {
             get
             {
-                return instance;
+                return new ImfManager();
+            }
+        }
+
+        /// <summary>
+        /// Dispose
+        /// </summary>
+        /// <param name="type">Dispose Type</param>
+        /// <since_tizen> 3 </since_tizen>
+        /// Please DO NOT use! This will be deprecated!
+        /// Dispose() method in Singletone classes (ex: FocusManager, StyleManager, VisualFactory, ImfManager, TtsPlayer, Window) is not required.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override void Dispose(DisposeTypes type)
+        {
+            if (disposed)
+            {
+                return;
+            }
+
+            if (type == DisposeTypes.Explicit)
+            {
+                //Called by User
+                //Release your own managed resources here.
+                //You should release all of your own disposable objects here.
+
+            }
+
+            //Release your own unmanaged resources here.
+            //You should not access any managed member here except static instance.
+            //because the execution order of Finalizes is non-deterministic.
```

Comment:
If you have a chance to change this please update it.

I think you want to write one sentence. If  so, please remove  "." in "//You should not access any managed member here except static instance.".

```
            //You should not access any managed member here except static instance  
            //because the execution order of Finalizes is non-deterministic.
```